app/controllers/seats_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class SeatsController < ApplicationController
+  before_action :set_seats, only: %i[show destroy]
+  # GET /seats
+  def index
+    seats = Seat.all
+
+    render json: seats
+  end
+
+  # GET /seats/1
+  def show
+    render json: @seat
+  end
+
+  # DELETE /seats/1
+  def destroy
+    seat = Seat.find(params[:id])
+    seat.destroy
```

Comment:
You don't have to return any JSON for this action, but consider that you should send information about success of this action, for instance - `head :no_content`